@here/harp-mapview/lib/MapView.ts

```diff
@@ -614,6 +614,31 @@ export interface MapViewOptions {
      * @default false.
      */
     synchronousRendering?: boolean;
+
+    /**
+     * Set initial center of the scene in geo coordinates.
+     * Longitude values outside of -180 and +180 are acceptable.
+     * Default is `new GeoCoordinates(25, 0, 30000000)`
+     */
+    geoCenter?: GeoCoordinates;
+
+    /**
+     * Set initial zoom level.
+     * Default is `12`
```

Comment:
Who wins when you set `geoCenter` with altitude and `zoomLevel` ?

Synchronize `geoCenter` and `zoomLevel` `@default` secstions so it's clearly visible who wins and where is actual default z:12 or d:3000000 ?

Reply:
The property `geoCenter` probably had the wrong name and was confusing. The newer `lookAt` API uses a `target`, i.e., the point that the camera focuses instead of setting position of the camera itself with `geoCenter`. For most use cases this is probably more useful. The altitude doesn't really have an impact on the initial zoom level. It is only configured by zoom level.